providers/themeProvider.tsx

```diff
@@ -8,9 +8,9 @@ import { THEME_STORAGE_KEY } from '@/next.constants.mjs';
 export const ThemeProvider: FC<PropsWithChildren> = ({ children }) => (
   <NextThemeProvider
     attribute="data-theme"
-    defaultTheme="dark"
+    defaultTheme="system"
```

Comment:
Though `defaultTheme="system"` marked as default in the [next-themes doc](https://github.com/pacocoursey/next-themes) it didn't work when I skipped it.